jira/server/scio_search/src/main/resources/vm/input.vm

```diff
@@ -1,13 +1,13 @@
 <html>
    <head>
-      <title>Configure Glean Search Plugin</title>
+      <title>Configure Glean Search Plugin Target URL</title>
```

Comment:
should we add "events" here? it may not be clear what the target here corresponds to. Maybe "Configure events target URL for Glean Search Plugin"?